### ISSUE TITLE
Let user choose Java for specific version

### ISSUE THREAD

**Issue by glowiak** [PolyMC/PolyMC#284]
### Role

I am tired of editing every instance to select java version

### Suggestion

Would be good to PolyMC not automatically download and manage java versions, but instead provide a menu, like Crystal Launcher does, where you enter the path for java for different MC versions (rd-1.12.2 (jdk8+lwjgl2), 1.13-1.16.5 (jdk8+lwjgl3) and 1.17+ (latest jdk and lwjgl3)).

### Benefit

So I can hit play after creating the instance, without editing the java path

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

It's somewhat different aproach than https://github.com/PolyMC/PolyMC/issues/7

**Comment by DioEgizio** [comment]
Yes that's a good idea as a *part* of #7, we should do #7 anyways but *optional*

**Comment by glowiak** [comment]
Yeah that's very good idea.

**Comment by piuvas** [comment]
Also, like DioEgizio mentioned in #7, it would require adding more paths manually if Minecraft ends up requiring a new java version. If that ends up being a roadblock, maybe another approach could be manually configuring multiple java "profiles" and being able to select them per instance, but that strays from the original idea of using the java check to automatically decide which version is used.
